### PR TITLE
Include configuration

### LINKED ISSUE #9: mb-config.json
Add `mb_config.json` to repo based on current version in `messagebroker-config`: https://github.com/DoSomething/messagebroker-config/blob/master/mb_config.json

### PR DESCRIPTION
**Pull request by sergiitk**
Include Message Broker configuration json into the repo.
Closes #9.

### CODE REVIEW

**Comment by deezone** [comment]
Should `config.json` have a more descriptive name? As `config.json` could it not be confused with the configuration of the module? How about `rabbitmq_config.json`? Perhaps also in it's own directory: `/config/rabbitmq_config.json`

**Comment by sergiitk** [comment]
Yes, I was in doubt about that. Thanks for the suggestion!

File: test/quicksiler-config-client.test.js [hunk 1]
```diff
@@ -16,48 +16,39 @@ describe('QuicksilverConfigClient', () => {
...
+  describe('getAllSettings()', () => {
     // Check getSettings method.
-    it('getAllSettings() should be exposed', () => {
+    it('should be exposed', () => {
       getConfigClient().getAllSettings.should.be.a.Function();
     });
 
     // Get all settings.
-    it('should return all settings in mb_config.json', () => {
+    it('should return all settings in the config file', () => {
```
Comment (deezone) [inline test/quicksiler-config-client.test.js]: ... in `rabbitmq_config` file.

File: test/quicksiler-config-client.test.js [hunk 2]
```diff
@@ -33,7 +33,7 @@ describe('QuicksilverConfigClient', () => {
     });
 
     // Check that config.json exists and is a file.
-    it('config file should exist', () => {
+    it('RabbitMQ config file be readable', () => {
```
Comment (deezone) [inline test/quicksiler-config-client.test.js]: RabbitMQ config file **must** be readable

File: test/quicksiler-config-client.test.js [hunk 3]
```diff
@@ -33,7 +33,7 @@ describe('QuicksilverConfigClient', () => {
     });
 
     // Check that config.json exists and is a file.
-    it('RabbitMQ config file be readable', () => {
+    it('RabbitMQ config should file be readable', () => {
```
Comment (deezone) [inline test/quicksiler-config-client.test.js]: "RabbitMQ config file should be readable"

Comment (sergiitk) [inline test/quicksiler-config-client.test.js]: Agh. thanks.

Comment (sergii-tkachenko) [inline test/quicksiler-config-client.test.js]: Agh. thanks.

**Pull request merged**